variables.tf

```diff
@@ -654,3 +654,7 @@ variable "use_test_date_as_onset_date" {
 variable "verify_rate_limit_secs" {
   description = "Time in seconds a user must wait before attempting to verify a one-time upload code"
 }
+variable "enable_quick_sight" {
```

Comment:
All the variables are in alpha order, eaiser to find
Since this is a new area, would create a new section - above this block with this flag

Reply:
done